packages/react-core/src/components/Label/Label.tsx

```diff
@@ -55,10 +55,12 @@ export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
   closeBtnAriaLabel?: string;
   /** Additional properties for the default close button. */
   closeBtnProps?: any;
-  /** Href for a label that is a link. If present, the label will change to an anchor element. */
+  /** Href for a label that is a link. If present, the label will change to an anchor element. This should not be passed in if the onClick prop is also passed in. */
   href?: string;
-  /** Flag indicating if the label is an overflow label */
+  /** Flag indicating if the label is an overflow label. This should not be passed in if the onClick prop is also passed in. */
   isOverflowLabel?: boolean;
+  /** Callback for when the label is clicked. This should not be passed in if the href or isOverflowLabel props are also passed in. */
```

Comment:
```suggestion
  /** Callback for when the label is clicked. This should not be passed in if the href prop is also passed in. */
```

Reply:
This should probably be changed to 'or isEditable prop` right?

Reply:
Oh yep, should be a swap instead of a removal